### PR TITLE
feat(metrics): add option for metrics in menu

### PR DESCRIPTION
**Pull request by mleeBoeing**
Admins and Org Stewards have a link to Metrics in the main menu.

### CODE REVIEW

**Comment by flesher1** [comment]
Looks good. :+1: 

Not really your problem, but I feel we could do with fewer sections/section titles in the menu. Its a little redundant and excessive as is. Maybe we could go with **Personalize** and **Manage** as the two sections and titles. This can be overlooked for the sake of this PR; just a note I wanted to make.

File: src/components/navBar/NavBar.js [hunk 1]
```diff
@@ -56,12 +58,18 @@ class NavBar extends React.Component {
...
     render() {
         var { isLoggedIn, notifications, unacknowledgedNotifications, profile } = this.props;
+        var showAdminOptions = false;
 
         var themeItems = Theme.VALUES.map(t => (
             <MenuItem key={t.value} text={t.label} icon={t.icon}
                 onClick={() => this.changeTheme(t.value)}
                 aria-label={`use ${t.label}`}/>
         ));
+        if (this.props.user) {
```
Comment (irtiza-rizvi) [inline src/components/navBar/NavBar.js]: @flesher1 might be a better person to ask, but I think you can use the `checkUserRoleAccess` helper function in `components/shared/CheckUserAccess`

**Comment by irtiza-rizvi** [comment]
:+1: works well.

Agree on the menu comment.  2.0 has Listing Management, Settings, and Metrics under the _Manage_ category, which looks cleaner.

**Comment by blynch11p** [comment]
To add on to what those guys said, I'd just stick with the 2.0 menu layout.
screenshot from 2018-08-28 07-29-03

**Comment by flesher1** [comment]
:+1: looks good

**Comment by blynch11p** [comment]
Two minor things.  

Make it `target="_blank"` on the metrics link to open the site in a new tab/window.

And, since you added user-level filter to AML Settings, only Admins (AML) can see that, not Org Stewards

**Comment by blynch11p** [comment]
good :+1:

**Pull request merged**